docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
     the modified CFS configuration. If more than one CFS configuration was
     modified, use the first one.
 
+## Optional: Configuring SAT to run on an external system over the CAN
+
+It is possible to configure SAT so that it can interact with CSM over the CAN. This allows SAT to run from a physical or
+virtual machine which is not a part of the CSM management cluster.
```

Comment:
If we add steps on installing SAT in a python virtualenv, we can say something like this (includes minor rewording/shortening):

```suggestion
SAT can optionally be installed and configured on an external system to interact with CSM over the CAN.
```